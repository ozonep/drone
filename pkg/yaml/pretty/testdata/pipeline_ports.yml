kind: pipeline
name: default

steps:
- name: redis
  pull: always
  detatch: true
  image: redis:latest
  ports:
  - 6379
  - port: 6379
    host: 6379
    protocol: TCP
