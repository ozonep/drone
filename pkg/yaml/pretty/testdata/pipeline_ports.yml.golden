---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: redis
  pull: always
  image: redis:latest
  ports:
  - 6379
  - port: 6379
    host: 6379
    protocol: TCP

...
